runtime/Makefile

```diff
@@ -8,8 +8,9 @@ MK_DIR := $(dir $(MK_ABSPATH))
 RT_BUILD_DIR?=$(MK_DIR)/build
 CMAKE_ARGS?=""
 CODE_COVERAGE?=OFF
-BUILD_TYPE?=Release
+BUILD_TYPE?=RelWithDebInfo
```

Comment:
Why the change here?

Reply:
What do you mean? Are you against this change?